choc-pizza.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>
+			Chocolate Pizza
+		</title>
+		<link href="reset-choc.css" type="text/css" rel="stylesheet" />
+		<link href="style-choc.css" type="text/css" rel="stylesheet" />
+	</head>
+	<body>
+		<header>
+			<div id="delicious">
+				Delicious
+			</div>
+			<div id="sub-delicious">
```

Comment:
This could be an h1. Remember, divs are dividers and have little semantic value.